pkg/config/config.go

```diff
@@ -114,6 +114,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("check_runners", int64(4))
 	config.BindEnvAndSetDefault("auth_token_file_path", "")
 	config.BindEnvAndSetDefault("bind_host", "localhost")
+	config.BindEnvAndSetDefault("health_port", int64(0))
```

Comment:
I think we should set the probe port to something different than 0, so that when using the agent in a different context than Kubernetes, users will not have the `port should be non-zero` error or have to add an extra parameter to have the health check. WDYT ?

Reply:
[checking healthPort > 0 in `start.go`](https://github.com/DataDog/datadog-agent/pull/2359/files#diff-1da27710e4127e711e81e8e4d4b390a5R147) guards us against that. No log line is displayed in the default configuration.